sqlmesh/core/context.py

```diff
@@ -205,22 +200,30 @@ def __init__(
         users: t.Optional[t.List[User]] = None,
     ):
         self.console = console or get_console()
-        self.path = Path(path).absolute()
-        if not self.path.is_dir():
-            raise ConfigError(f"{path} is not a directory")
 
-        self.config = self._load_config(config or "config")
+        self._sqlmesh_path = Path.home() / ".sqlmesh"
 
-        self.physical_schema = physical_schema or self.config.physical_schema or "sqlmesh"
-        self.snapshot_ttl = snapshot_ttl or self.config.snapshot_ttl or c.DEFAULT_SNAPSHOT_TTL
-        self.dag: DAG[str] = DAG()
+        _paths = [
+            Path(path).absolute() for path in ([paths] if isinstance(paths, str) else list(paths))
+        ]
+
+        for path in _paths:
+            if not path.is_dir():
+                raise ConfigError(f"{path} is not a directory")
 
+        self.configs = self._load_configs(config or "config", _paths)
+
+        self.dag: DAG[str] = DAG()
         self._models: UniqueKeyDict[str, Model] = UniqueKeyDict("models")
         self._audits: UniqueKeyDict[str, Audit] = UniqueKeyDict("audits")
         self._macros: UniqueKeyDict[str, ExecutableOrMacro] = UniqueKeyDict("macros")
         self._hooks: UniqueKeyDict[str, hook] = UniqueKeyDict("hooks")
 
+        self.path, self.config = t.cast(t.Tuple[Path, Config], next(iter(self.configs.items())))
```

Comment:
Didn't we decide not to take the first config?

Reply:
i said i was going to do first config temporarily, and we can later do a master config